app/models/widget.rb

```diff
@@ -0,0 +1,16 @@
+class Widget < ActiveRecord::Base
+  enum widget_type: Constants::WIDGET_TYPES
```

Comment:
Don't we need just one `enum`, which holds all of the widget types, just as I did it? If so, then yes, if not, why not? :)